### PR TITLE
[documentation] Use the documented coding style in the examples

### PR DESCRIPTION
**Pull request by bbolli**
##### Checklist

- [x] documentation is added or updated
- [x] CLA: trivial

##### Description of change

Adjust brace placement and whitespace after keywords according to
https://www.openssl.org/policies/codingstyle.html.

### CODE REVIEW

**Comment by mattcaswell** [comment]
I start to feel uncomfortable when I see changes of this size coming in under CLA:trivial. While an argument could be made for it, I think it would be better to have a CLA on file for this change.

**Comment by bbolli** [comment]
Don't you agree that the changes are really trivial?

**Comment by mattcaswell** [comment]
When looked at individually: yes. When looked as a collection there has clearly been some work put into this. I prefer us to err on the side of caution. What appears as trivial to us as technical people is not always trivial to a lawyer:
https://majadhondt.wordpress.com/2012/05/16/googles-9-lines/

It does not take much to stray over the line...and it is very easy to file a CLA and thus avoid the problem.

**Comment by bbolli** [comment]
OK, I'll send the CLA in a few minutes. Shall I resubmit this patch later or do you put it on hold?

**Comment by mattcaswell** [comment]
It's just on hold with the "Need CLA" label until such time as we receive your CLA, and then we can take the label off again. No need to resubmit.

**Comment by bbolli** [comment]
Bump! I think you've got my CLA, haven't you?

**Comment by richsalz** [comment]
yes, thanks.  now just needs team review.

File: doc/man3/EVP_EncryptInit.pod [hunk 1]
```diff
@@ -540,7 +540,7 @@ for certain common S/MIME ciphers (RC2, DES, triple DES) in CBC mode.
 Encrypt a string using IDEA:
 
  int do_crypt(char *outfile)
-        {
+ {
         unsigned char outbuf[1024];
```
Comment (levitte) [inline doc/man3/EVP_EncryptInit.pod]: 4 space indent, not 8, throughout this example

File: doc/man3/BIO_f_base64.pod [hunk 2]
```diff
@@ -65,7 +65,7 @@ data to standard output:
  bio = BIO_new_fp(stdin, BIO_NOCLOSE);
  bio_out = BIO_new_fp(stdout, BIO_NOCLOSE);
  BIO_push(b64, bio);
- while((inlen = BIO_read(b64, inbuf, 512)) > 0)
+ while ((inlen = BIO_read(b64, inbuf, 512)) > 0)
         BIO_write(bio_out, inbuf, inlen);
```
Comment (levitte) [inline doc/man3/BIO_f_base64.pod]: Make that a 4 space indent

File: doc/man3/ENGINE_add.pod [hunk 3]
```diff
@@ -474,7 +474,7 @@ boolean success or failure.
      ENGINE *e = ENGINE_by_id(engine_id);
      if (!e) return 0;
      while (pre_num--) {
-         if(!ENGINE_ctrl_cmd_string(e, pre_cmds[0], pre_cmds[1], 0)) {
+         if (!ENGINE_ctrl_cmd_string(e, pre_cmds[0], pre_cmds[1], 0)) {
              fprintf(stderr, "Failed command (%s - %s:%s)\n", engine_id,
                  pre_cmds[0], pre_cmds[1] ? pre_cmds[1] : "(NULL)");
```
Comment (levitte) [inline doc/man3/ENGINE_add.pod]: Our style aligns function call arguments so they're all right of the opening parenthesis

File: doc/man3/EVP_PKEY_keygen.pod [hunk 4]
```diff
@@ -133,7 +133,7 @@ Example of generation callback for OpenSSL public key implementations:
  EVP_PKEY_CTX_set_app_data(ctx, status_bio);
 
  static int genpkey_cb(EVP_PKEY_CTX *ctx)
-        {
+ {
         char c = '*';
```
Comment (levitte) [inline doc/man3/EVP_PKEY_keygen.pod]: 4 space indent, not 8

File: doc/man3/SSL_CTX_set_info_callback.pod [hunk 5]
```diff
@@ -110,7 +110,7 @@ The following example callback function prints state strings, information
 about alerts being handled and error messages to the B<bio_err> BIO.
 
  void apps_ssl_info_callback(SSL *s, int where, int ret)
-        {
+ {
         const char *str;
```
Comment (levitte) [inline doc/man3/SSL_CTX_set_info_callback.pod]: 4 space indent

File: doc/man3/X509_NAME_get_index_by_NID.pod [hunk 6]
```diff
@@ -75,25 +75,23 @@ Process all entries:
  int i;
  X509_NAME_ENTRY *e;
 
- for (i = 0; i < X509_NAME_entry_count(nm); i++)
-        {
+ for (i = 0; i < X509_NAME_entry_count(nm); i++) {
         e = X509_NAME_get_entry(nm, i);
         /* Do something with e */
```
Comment (levitte) [inline doc/man3/X509_NAME_get_index_by_NID.pod]: 4 space indent

File: doc/man3/X509_NAME_get_index_by_NID.pod [hunk 7]
```diff
@@ -75,25 +75,23 @@ Process all entries:
...
  X509_NAME_ENTRY *e;
 
- for (;;)
-        {
+ for (;;) {
         lastpos = X509_NAME_get_index_by_NID(nm, NID_commonName, lastpos);
         if (lastpos == -1)
                 break;
         e = X509_NAME_get_entry(nm, lastpos);
         /* Do something with e */
```
Comment (levitte) [inline doc/man3/X509_NAME_get_index_by_NID.pod]: 4 space indent

**Comment by bbolli** [comment]
OK, I'll resubmit later.

**Comment by bbolli** [comment]
There's still more to come, but it'll have to wait until Friday.

**Comment by bbolli** [comment]
Pushing a rebased branch after the merge of https://github.com/openssl/openssl/pull/1618, which caused conflicts

**Comment by bbolli** [comment]
Bump @levitte for team review

File: doc/man3/BIO_f_base64.pod (hunk 2 above)
Comment (levitte) [inline doc/man3/BIO_f_base64.pod]: Still make that a 4 space indent...

File: doc/man3/CRYPTO_THREAD_run_once.pod [hunk 8]
```diff
@@ -142,9 +142,9 @@ You can find out if OpenSSL was configured with thread support:
 
  #include <openssl/opensslconf.h>
  #if defined(OPENSSL_THREADS)
-   // thread support enabled
+     // thread support enabled
  #else
-   // no thread support
+     // no thread support
```
Comment (levitte) [inline doc/man3/CRYPTO_THREAD_run_once.pod]: Would you mind changing those to C comments (i.e. `/* stuff */`)?

File: doc/man3/EVP_BytesToKey.pod [hunk 9]
```diff
@@ -44,7 +44,7 @@ defined in PKCS#5v2.1 and provided by PKCS5_PBKDF2_HMAC.
 The key and IV is derived by concatenating D_1, D_2, etc until
 enough data is available for the key and IV. D_i is defined as:
 
-        D_i = HASH^count(D_(i-1) || data || salt)
+ D_i = HASH^count(D_(i-1) || data || salt)
```
Comment (levitte) [inline doc/man3/EVP_BytesToKey.pod]: This isn't exactly standard example code.  I would rather have the original indentation kept.

File: doc/man3/BIO_f_base64.pod (hunk 2 above)
Comment (bbolli) [inline doc/man3/BIO_f_base64.pod]: Got'em all now...

File: doc/man3/BIO_f_md.pod [hunk 10]
```diff
@@ -118,15 +119,16 @@ outputs them. This could be used with the examples above.
...
-        mdtmp = BIO_find_type(mdtmp, BIO_TYPE_MD);
-        if (!mdtmp) break;
-        BIO_get_md(mdtmp, &md);
-        printf("%s digest", OBJ_nid2sn(EVP_MD_type(md)));
-        mdlen = BIO_gets(mdtmp, mdbuf, EVP_MAX_MD_SIZE);
-        for (i = 0; i < mdlen; i++) printf(":%02X", mdbuf[i]);
-        printf("\n");
-        mdtmp = BIO_next(mdtmp);
+     EVP_MD *md;
+     mdtmp = BIO_find_type(mdtmp, BIO_TYPE_MD);
```
Comment (FdaSilvaYY) [inline doc/man3/BIO_f_md.pod]: blank line after decl.

File: doc/man3/BIO_s_file.pod [hunk 11]
```diff
@@ -92,24 +92,29 @@ Alternative technique:
 
  BIO *bio_out;
```
Comment (FdaSilvaYY) [inline doc/man3/BIO_s_file.pod]: blank line after decl.

**Comment by bbolli** [comment]
@FdaSilvaYY what else do you need?

**Comment by mattcaswell** [comment]
Ping @levitte

**Comment by bbolli** [comment]
Rebased and repushed after changes to `doc/man3/EVP_EncryptInit.pod`

**Comment by levitte** [comment]
@openssl/team, someone else to review as well.

**Comment by bbolli** [comment]
Rebased and repushed after bf0d560938

**Comment by mattcaswell** [comment]
Pushed. Thanks for your contribution. Sorry it took so long to get through the process.

**Pull request closed without merging**